src/routes/docs/advanced/self-hosting/sites/+page.markdoc

```diff
@@ -0,0 +1,133 @@
+---
+layout: article
+title: Sites
+description: Harness the full power of self-hosted sites with Appwrite. Explore site deployment, management, and integration in your self-hosted environment.
+---
+
+{% partial file="configure-github-app.md" /%}
+
+# Configure Sites runtimes {% #sites %}
+
+Not all site runtimes are enabled by default. Enable the runtimes that you need and disable unused runtimes to save disk space on your server.
+To enable a runtime, add it to the `_APP_SITES_RUNTIMES` environment variable as a comma-separated list.
+
+The example below would enable Static, Node.js 22, and Flutter 3.29 runtimes.
+
+```bash
+_APP_SITES_RUNTIMES=static-1,node-22,flutter-3.29
+```
+
+You can also configure the maximum timeout that can be set on individual Appwrite Sites. The maximum configurable timeout can be increased by changing the `_APP_SITES_TIMEOUT` environment variable. This environment variable changes the configurable maximum but does not alter existing configurations of individual sites.
+
+{% partial file="update-variables.md" /%}
+
+# Add an apex domain for Sites {% #root-domain %}
+
+Appwrite allows you to add an apex domain to your instance's configuration so that you can publicly expose your Sites as subdomains .
+To add an apex domain, you must first configure either one of the following environment variables in your `.env` file:
+
+- `_APP_DOMAIN_TARGET_A`: Set this to the IPv4 address of your server.
+- `_APP_DOMAIN_TARGET_AAAA`: Set this to the IPv6 address of your server.
+
+Ensure that the `_APP_DOMAIN_SITES` environment variable is set to `<OPTIONAL_SUBDOMAIN>.<APEX_DOMAIN>`, where `<APEX_DOMAIN>` is the domain you want to use for your sites. For example, if your apex domain is `myapp.com`, you can set `_APP_DOMAIN_SITES` to `myapp.com` or `sites.myapp.com` (or any other such subdomain).
+
+Next, head to your DNS provider and create an A or AAAA record for your root domain pointing to the server's IP address. This will allow Appwrite to serve your sites from the apex domain. 
```

Comment:
Might be a good note to remove all existing A and AAAA records.
I often do the mistake of adding A record, but then my domain doesnt work, because my old records had AAAA too, and now it missbehaves. Removing old AAAA solves it for me